coordinator/src/services/translation/response/release/release-medication-request.ts

```diff
@@ -187,17 +185,20 @@ export function createSnomedCodeableConcept(code: hl7V3.SnomedCode): fhir.Codeab
   return fhir.createCodeableConcept("http://snomed.info/sct", code._attributes.code, code._attributes.displayName)
 }
 
-export function createDosage(
-  dosageInstructions: hl7V3.DosageInstructions,
-  additionalInstructions: string
-): fhir.Dosage {
-  const dosage: fhir.Dosage = {
-    text: dosageInstructions.value._text
+export function createNote(additionalInstructions: string): Array<fhir.Annotation> {
+  if (!additionalInstructions) {
+    return undefined
   }
-  if (additionalInstructions) {
-    dosage.patientInstruction = additionalInstructions
+
+  return [{
+    text: additionalInstructions
```

Comment:
will spine ever return multiple patient instructions? (not sure how this would be represented in v3)

Reply:
No, there's only a single HL7 V3 field for these